driver/src/main/com/mongodb/reactivestreams/client/gridfs/GridFSBuckets.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.reactivestreams.client.gridfs;
+
+import com.mongodb.reactivestreams.client.MongoDatabase;
+
+/**
+ * A factory for GridFSBucket instances.
+ *
+ * @since 1.3
+ */
+public final class GridFSBuckets {
+
+    /**
+     * Create a new GridFS bucket with the default {@code 'fs'} bucket name
+     *
+     * @param database the database instance to use with GridFS
+     * @return the GridFSBucket
+     */
+    public static GridFSBucket create(final MongoDatabase database) {
+        return new GridFSBucketImpl(com.mongodb.async.client.gridfs.GridFSBuckets.create(database.getAsyncMongoDatabase()));
```

Comment:
I can't think of a clean way to access the wrapped MongoDatabase without explicitly adding a method to the interface surfacing it.